addon/modifiers/enable-input-autofocus.ts

```diff
@@ -0,0 +1,35 @@
+// @ts-ignore
+import { setModifierManager, capabilities } from '@ember/modifier';
+
+export default setModifierManager(
+  () => ({
+    capabilities: capabilities('3.22'),
+
+    createModifier() {
```

Comment:
no method has a return value, I'm guessing this is also the case in other modifiers.
Maybe add some ?